main/settings.py

```diff
@@ -597,6 +597,7 @@ def log_render_extra_context(record):
 
 ERP_API_ENDPOINT = env("ERP_API_ENDPOINT")
 ERP_API_SUBSCRIPTION_KEY = env("ERP_API_SUBSCRIPTION_KEY")
+ERP_API_MAX_REQ_TIMEOUT = 60 * 2  # 2 minutes
```

Comment:
@susilnem Please ensure the request timeout is set to under 2 minutes, as this is the maximum duration allowed  in the Gunicorn configuration. Ideally we wan't it be less than a minute.